arcgis-ios-sdk-samples/Scenes/Display 3D labels in scene/Display3DLabelsSceneViewController.swift

```diff
@@ -0,0 +1,96 @@
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import UIKit
+import ArcGIS
+
+class Display3DLabelsSceneViewController: UIViewController {
+    @IBOutlet var sceneView: AGSSceneView! {
+        didSet {
+            // Set the scene to the scene view.
+            sceneView.scene = makeScene()
+        }
+    }
+    
+    func makeScene() -> AGSScene {
+        let scene = AGSScene(
+            item: AGSPortalItem(
+                portal: AGSPortal.arcGISOnline(withLoginRequired: false),
+                itemID: "850dfee7d30f4d9da0ebca34a533c169")
+        )
+        // Load the scene.
+        scene.load { [weak self] error in
+            // Get the feature layer.
+            if let layers = scene.operationalLayers as? [AGSGroupLayer],
+               let groupLayer = layers.first(where: { $0.name == "Gas" }),
+               let gasFeatureLayer = groupLayer.layers.firstObject as? AGSFeatureLayer {
+                do {
+                    guard let labelDefinition = try self?.makeLabelDefinition() else { return }
+                    // Enable labels on the feature layer.
+                    gasFeatureLayer.labelsEnabled = true
+                    // Add the label definition to the layer.
+                    gasFeatureLayer.labelDefinitions.add(labelDefinition)
+                } catch {
+                    // If failure to make a label definition, present an error.
+                    print(error)
+                }
+            }
+        }
+        return scene
+    }
+    
+    func makeLabelDefinition() throws -> AGSLabelDefinition? {
+        // Make and stylize the text symbol.
+        let textSymbol = AGSTextSymbol()
+        textSymbol.angle = 0
+        textSymbol.backgroundColor = .clear
+        textSymbol.outlineColor = .white
+        textSymbol.color = .orange
+        textSymbol.haloColor = .white
+        textSymbol.haloWidth = 2
+        textSymbol.horizontalAlignment = .center
+        textSymbol.verticalAlignment = .middle
+        textSymbol.isKerningEnabled = false
+        textSymbol.offsetX = 0
+        textSymbol.offsetY = 0
+        textSymbol.fontDecoration = .none
+        textSymbol.size = 14
+        textSymbol.fontStyle = .normal
+        textSymbol.fontWeight = .normal
+        let textSymbolJSON = try textSymbol.toJSON()
+
+        // Make a JSON object.
+        let labelJSONObject: [String: Any] = [
+            "labelExpressionInfo": [
+                "expression": "$feature.INSTALLATIONDATE"
```

Comment:
This question will leave to Phil's decision: Maybe this expression string can be passed as an argument to the method.

Because it makes more sense to me that we can create a label definition (JSON/`AGSLabelDefinition` object) by specifying a label expression, and use the default symbol. 

It would be even greater if we can also pass a text symbol  to this method, and create the text symbol outside the method with a factory method/closure block. But this might be trivial for the sample.